styles/style.css

```diff
@@ -0,0 +1,75 @@
+.container {
+   
+    width: 1900px;
+    height: 900px;
+    display: grid; 
+    grid-template-columns: 1fr 1fr ;
+    grid-template-rows: 1fr 1fr 1fr ;
+    
+}
+
+.flex-container {
+   
+    width: 1900x;
```

Comment:
A typo in the units `x` rather than `px` prevents this rule from taking effect. But we should avoid setting huge widths in general, as it can require visitors to scroll horizontally to read text, which reduces ease of use. Try to fit to the user's screen size, and restrict yourself to vertical scrolling.